indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -407,10 +403,42 @@ $i-box-padding: 10px;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        /* Sortable row with handle on the left */
```

Comment:
So maybe we should enable the rule `no-css-comments` in the sass-lint file. It seems to fit exactly with your suggestion! Info: https://github.com/sasstools/sass-lint/blob/master/docs/rules/no-css-comments.md